src/sbt-test/sbt-catalysts/smoke/test

```diff
@@ -1,5 +1,5 @@
 # check if the file gets created
 > project coreJVM
 > compile
-> project smoke
-> publishLocal
+> rootJS/publishLocal
```

Comment:
@kailuowang I had to change this because the root project isn't called "smoke" anymore in sbt 1.4.